src/machines/java/com/enderio/machines/common/blockentity/base/MachineBlockEntity.java

```diff
@@ -394,8 +385,10 @@ protected void clearCaches() {
 
     protected void populateCaches(Direction direction, @Nullable BlockEntity neighbor) {
         if (neighbor != null) {
-            itemHandlerCache.put(direction, addInvalidationListener(neighbor.getCapability(CapabilityItemHandler.ITEM_HANDLER_CAPABILITY, direction.getOpposite())));
-            fluidHandlerCache.put(direction, addInvalidationListener(neighbor.getCapability(CapabilityFluidHandler.FLUID_HANDLER_CAPABILITY, direction.getOpposite())));
+            itemHandlerCache.put(direction, addInvalidationListener(
+                    neighbor.getCapability(CapabilityItemHandler.ITEM_HANDLER_CAPABILITY, direction.getOpposite())));
+            fluidHandlerCache.put(direction, addInvalidationListener(
+                    neighbor.getCapability(CapabilityFluidHandler.FLUID_HANDLER_CAPABILITY, direction.getOpposite())));
```

Comment:
Again, this is because I am coding with VSCode, my apologies. I did format it with IntelliJ which gave something different to both, so I have left that in.